src/App.tsx

```diff
@@ -31,6 +47,20 @@ export default function App() {
             path="/activities"
             element={<ActivitiesList activities={ACTIVITIES_MOCK} />}
           />
+          <Route
+            path="/signup"
+            element={<AccountCreation onAccountCreate={handleAccountCreate} />}
+          />
+          <Route
+            path="/account"
+            element={
+              isAuthenticated ? (
+                <AuthenticatedContent username={""} email={""} password={""} />
```

Comment:
We can remove `password` from here too

```suggestion
                <AuthenticatedContent username={""} email={""} />
```